browser_patches/firefox/patches/bootstrap.diff

```diff
@@ -2819,7 +2819,22 @@ index 0000000000000000000000000000000000000000..37ab5f56739cfd16200a4ada9f4cf834
 +      .getService(Ci.nsIAccessibilityService);
 +    const document = this._frameTree.mainFrame().domWindow().document;
 +    const docAcc = service.getAccessibleFor(document);
++    const content = this._session.mm().content;
 +
++    async function waitForRoot() {
++      let callback;
++      const result = new Promise(f => callback = f);
++      const poll = () => {
++        if (!docAcc.firstChild && content.document.body.hasChildNodes()) {
```

Comment:
Is it always racing against a blank document? I expected it to be racing against some previous cached DOM state, which happens to be blank in our tests.

This will also stall forever on about:blank.